exercises/change/change_test.go

```diff
@@ -0,0 +1,34 @@
+package change
+
+import (
+	"reflect"
+	"testing"
+)
+
+const targetTestVersion = 1
+
+func TestTestVersion(t *testing.T) {
+	if testVersion != targetTestVersion {
+		t.Fatalf("Found testVersion = %v, want %v", testVersion, targetTestVersion)
+	}
+}
+
+func TestChange(t *testing.T) {
+	for _, tc := range testCases {
+		actual := Change(tc.coins, tc.target)
+		if !reflect.DeepEqual(actual, tc.expected) {
+			t.Fatalf("%s : Change(%v, %d): expected %v, actual %v",
```

Comment:
While trying this out I changed some code which gave me this failure message:

```
change_test.go:21: no coins make 0 change : Change([1 5 10 21 25], 0): expected [], actual []
```
which reminded me of this issue https://github.com/exercism/xgo/issues/414

though changing `%v` to `%q` didn't help, `%#v` did, giving:

```
change_test.go:21: no coins make 0 change : Change([1 5 10 21 25], 0): expected []int{}, actual []int(nil)
```
I'm not sure if it's worth changing this?